evap/staff/forms.py

```diff
@@ -183,16 +183,25 @@ def validate_unique(self):
 
 
 class ContributionForm(forms.ModelForm, BootstrapMixin):
+    responsibility = forms.ChoiceField(widget=forms.RadioSelect(), choices=Contribution.RESPONSIBILITY_CHOICES)
+
     class Meta:
         model = Contribution
-        fields = "__all__"
+        fields = ('course', 'contributor', 'questionnaires', 'order', 'responsibility', 'comment_visibility')
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['contributor'].widget.attrs['class'] = 'form-control'
 
+        if self.instance.responsible:
+            self.initial['responsibility'] = Contribution.IS_RESPONSIBLE
+        elif self.instance.can_edit:
+            self.initial['responsibility'] = Contribution.IS_EDITOR
+        else:
+            self.initial['responsibility'] = Contribution.IS_CONTRIBUTOR
```

Comment:
From the documentation: `Note that if a Field defines initial and you include initial when instantiating the Form, then the latter initial will have precedence.`
Otherwise they do the same thing.